donkeycar/management/base.py

```diff
@@ -173,11 +173,12 @@ def run(self, args):
         
         print("Finding your car's IP address...")
         cmd = "sudo nmap -sP " + ip + "/24 | awk '/^Nmap/{ip=$NF}/B8:27:EB/{print ip}'"
+        cmdRPi4 = "sudo nmap -sP " + ip + "/24 | awk '/^Nmap/{ip=$NF}/DC:A6:32/{print ip}'"
         print("Your car's ip address is:" )
         os.system(cmd)
```

Comment:
I see the old command is hanging and not returning on RPi4, whereas the new one works. Can you add a flag that only one gets run and enable the new command by default ?